uikit-dz/uikit-dz/ChooseCoffeeViewController.swift

```diff
@@ -0,0 +1,237 @@
+        label.textColor = .black
+        return label
+    }()
+
+    lazy var ingredientsTypeButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 195, y: 482, width: 165, height: 165))
+        button.backgroundColor = #colorLiteral(red: 0.9728776813, green: 0.9728776813, blue: 0.9728776813, alpha: 1)
+        button.setImage(UIImage(named: "plus"), for: .normal)
+        button.addTarget(self, action: #selector(ingredientsTypeButtonPressed), for: .touchUpInside)
+        button.layer.cornerRadius = 15
+        return button
+    }()
+
+    let ingredientsTypeLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 215, y: 595, width: 135, height: 50))
+        label.font = UIFont(name: "Verdana", size: 16)
+        label.font = .systemFont(ofSize: 16, weight: .regular)
+        label.text = "Дополнительные ингредиенты"
+        label.numberOfLines = 2
+        label.textAlignment = .center
+        label.textColor = .black
+        return label
+    }()
+
+    let priceLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 15, y: 669, width: 345, height: 30))
+        label.textAlignment = .right
+        label.text = "Цена - 200 руб"
+        label.font = UIFont(name: "Verdana", size: 18)
+        label.font = .systemFont(ofSize: 18, weight: UIFont.Weight(700))
+        return label
+    }()
+
+    var coffeePrice = 200
+    var additions: [String] = []
+
+    lazy var orderButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 20, y: 717, width: 345, height: 53))
+        button.backgroundColor = #colorLiteral(red: 0.4395369291, green: 0.7734851837, blue: 0.8083773255, alpha: 1)
+        button.addTarget(self, action: #selector(orderButtonPressed), for: .touchUpInside)
+        button.layer.cornerRadius = 15
+        button.setTitle("Заказать", for: .normal)
+        button.titleLabel?.font = .systemFont(ofSize: 18, weight: UIFont.Weight(700))
+        return button
+    }()
+
+    lazy var coffeeTypeVC: CoffeeTypeViewController = {
+        let vc = CoffeeTypeViewController()
+        vc.delegate = self
+        return vc
+    }()
+
+    lazy var ingredientsVC: IngredientsViewController = {
+        let vc = IngredientsViewController()
+        vc.delegate = self
+        return vc
+    }()
+
+    // MARK: - Initializer
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupUI()
+    }
+
+    // MARK: - Private methods
+
+    private func setupUI() {
+        view.backgroundColor = .white
+        view.addSubview(topView)
+        view.addSubview(topImageView)
+        view.addSubview(segmentCoffee)
+        view.addSubview(configurationLabel)
+        view.addSubview(coffeeTypeButton)
+        view.addSubview(coffeeTypeLabel)
+        view.addSubview(ingredientsTypeButton)
+        view.addSubview(ingredientsTypeLabel)
+        view.addSubview(priceLabel)
+        view.addSubview(orderButton)
+
+        let leftBar = UIBarButtonItem(
```

Comment:
```suggestion
        let leftBarButtonItem = UIBarButtonItem(
```
и вынеси ко всем Ui все бары на верх